docs/en/observability/profiling-index-lifecycle-management.asciidoc

```diff
@@ -51,10 +51,9 @@ Complete the following steps to configure a custom index lifecycle policy.
 [[profiling-ilm-custom-policy-create-policy]]
 === Step 1: Create an index lifecycle policy
 
-. Navigate to **{stack-manage-app}** → **Index Lifecycle Policies**.
-
+. To open **Index Lifecycle Policies**, find **{stack-manage-app}** in the main menu or use the {kibana-ref}/introduction.html#kibana-navigation-search[global search field].

```

Comment:
I think you have removed all occurrences of {stack-manage-app}... in such case I would suggest:

```suggestion
. To open **Index Lifecycle Policies**, find **Stack Management** in the main menu or use the {kibana-ref}/introduction.html#kibana-navigation-search[global search field].

```

Reply:
Good catch. My main reason for avoiding the variable is that I want to be consistent in the approach we take so that we can do a search/replace when someone more important than me decides the guidelines we're following are awkward. :-) :-) So it's good for us to be consistent. I'm also starting to doubt the value of using vars when we use them so inconsistently.